app/_kong_plugins/openid-connect/index.md

```diff
@@ -37,6 +35,541 @@ search_aliases:
   - openid-connect
   - idp
   - identity provider
+
+tags:
+  - authentication
+
+related_resources:
+  - text: Authentication in {{site.base_gateway}}
+    url: /gateway/authentication/
 ---
 
-## Overview
+The OpenID Connect (OIDC) plugin lets you integrate {{site.base_gateway}} with an
+identity provider (IdP). This plugin can be used to implement
+Kong as a proxying [OAuth 2.0](https://tools.ietf.org/html/rfc6749) resource server 
+(RS) and as an OpenID Connect relying party (RP) between the client and the upstream service.
+
+## What does OpenID Connect do?
+
+OpenID Connect provides a way to form a **federation** with **identity providers (IdPs)**. 
+Identity providers are third parties that store account credentials. 
+If the identity provider authenticates a user, the application trusts that provider and allows access to the user. 
+The identity provider bears some responsibility for protecting the user’s credentials and ensuring authenticity, 
+so applications no longer need to on their own.
+
+Besides delegating responsibility to an identity provider, OpenID Connect also makes single sign-on possible 
+without storing any credentials on a user’s local machine.
+
+Enterprises may want to manage access control for many applications from one central system of record. 
+For example, they may want employees to be able to access many different applications using their email address and password. 
+They may want to also change access (for example, if an employee leaves or changes roles) from one central point. 
+OpenID Connect addresses this challenge by providing a way for many different applications to authenticate users through the 
+same third-party identity provider.
+
+## What does Kong’s OpenID Connect plugin do?
+
+Just as OpenID Connect enables developers to offload authentication to another party, {{site.base_gateway}} enables developers to 
+separate entire processes from their applications. Instead of needing to manually write the code for OpenID Connect 
+within a service, developers can place {{site.base_gateway}} in front of the upstream service and have {{site.base_gateway}} handle authentication. 
+This separation allows developers to focus on the business logic within their application, easily swap out services while preserving authentication at the front door, and effortlessly spread the same authentication to new services.
```

Comment:
```suggestion
The Open ID Connect plugin enables you to integrate Open ID Connect with {{site.base_gateway}} without having to write custom integrations. {{site.base_gateway}} enables developers to  separate entire processes from their applications. Instead of needing to manually write the code for OpenID Connect within a service, developers can place {{site.base_gateway}} in front of the upstream service and have {{site.base_gateway}} handle authentication. 
This separation allows developers to focus on the business logic within their application, easily swap out services while preserving authentication at the front door, and effortlessly spread the same authentication to new services.
```